packages/sourcecred/src/api/lib/node.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+
+// Extended exports for calling SourceCred code programmatically
+// in a node environment. Both the structure and the contents of
+// this API are experimental and subject to change.
+
+// import {GithubPlugin} from "../../plugins/github/plugin";
+// import {DiscordPlugin} from "../../plugins/discord/plugin";
+// import {DiscoursePlugin} from "../../plugins/discourse/plugin";
+// import {InitiativesPlugin} from "../../plugins/initiatives/plugin";
+
+import {LocalInstance} from "../../api/instance/localInstance";
```

Comment:
This import is what's causing your error, whether it's added to the api object or not on line 18.

I got the un-minified stack trace by setting `process.env.NODE_ENV = 'development'` in config/webpack.config.api.js:
```node
❯ node
Welcome to Node.js v14.17.0.
Type ".help" for more information.
> test = require('./dist/server/api.js')
Uncaught TypeError: Cannot read property 'indexOf' of undefined
    at Function.getFileName (webpack://sourcecred//home/kevin/Projects/sourceCred/sc-core/node_modules/bindings/bi
ndings.js?:178:16)
    at bindings (webpack://sourcecred//home/kevin/Projects/sourceCred/sc-core/node_modules/bindings/bindings.js?:8
2:48)
    at eval (webpack://sourcecred//home/kevin/Projects/sourceCred/sc-core/node_modules/better-sqlite3/lib/database
.js?:9:83)
    at Object.../../node_modules/better-sqlite3/lib/database.js (/home/kevin/Projects/sourceCred/sc-core/packages/
sourcecred/dist/server/api.js:409:1)
    at __webpack_require__ (/home/kevin/Projects/sourceCred/sc-core/packages/sourcecred/dist/server/api.js:21:30)
    at eval (webpack://sourcecred//home/kevin/Projects/sourceCred/sc-core/node_modules/better-sqlite3/lib/index.js
?:2:18)
    at Object.../../node_modules/better-sqlite3/lib/index.js (/home/kevin/Projects/sourceCred/sc-core/packages/sou
rcecred/dist/server/api.js:421:1)
    at __webpack_require__ (/home/kevin/Projects/sourceCred/sc-core/packages/sourcecred/dist/server/api.js:21:30)
    at eval (webpack://sourcecred/./src/plugins/github/plugin.js?:3:72)
    at Module../src/plugins/github/plugin.js (/home/kevin/Projects/sourceCred/sc-core/packages/sourcecred/dist/ser
ver/api.js:5411:1)
```

It looks like it's coming from better-sqlite. My guess is it's looking to import some file immediately? Not entirely sure but if you set build mode to development you might be able to track it down